### ISSUE TITLE
ActiveForm statements are not sitemapped

### ISSUE THREAD

**Issue by bgyori** [sorgerlab/indra#65]
```
 ActiveForm(MAPK1(mods: (phosphorylation, T, 185)), kinase, True),
 ActiveForm(MAPK1(mods: (phosphorylation, Y, 185)), kinase, True),
```

**Comment by bgyori** [comment]
This might actually be an issue in RAS 220 specifically and not in the sitemapper itself.

**Comment by bgyori** [comment]
Okay, so the actual issue here is that if there is no UniProt ID attached to the agent, we just give up. I'll make sure that HGNC ID is sufficient for site mapping.

**Comment by bgyori** [comment]
Fixed in fbc8a2a3d48c2c215c845d3331951930122d65db, now HGNC ID is sufficient for site mapping. If UniProt ID is given, it takes precedence.
